google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreOptions.java

```diff
@@ -87,6 +107,9 @@ public Builder setTransportOptions(TransportOptions transportOptions) {
 
     @Override
     public DatastoreOptions build() {
+      if (this.openTelemetryOptions == null) {
```

Comment:
instead of this, you can set `openTelemetryOptions` to the below as the default value, and then allow the setter to override it for a customization.

or, this logic should probably move to the constructor of DatastoreOptions rather than in the `build()` method.

Reply:
Looks like this was a redundant defensive check - removed it as it's already handled in the constructor.